### DISCUSSION TITLE
✅ Authentication with private key only?

### DISCUSSION THREAD

**Discussion by svenreppin** [isontheline/pro.webssh.net#728] [General]
hi, is there a way to authenticate with private key and without password only?

**Comment by isontheline** [comment]
Hi 👋

Could you explain me more?
You can already use a PKEY without any password or passphrase.

Are you encountering any error?

Have a nice day ☀️

**Comment by svenreppin** [comment]
hi, thank you very much for your prompt answer!!! my server's sshd is configured with `PasswordAuthentication no` and so i am not able to login. there is no further notification nor any message

**Reply by isontheline** [reply]
Hi 😉

But if you use a private key is there an error?

Have you submitted your pkey inside WebSSH?

**Comment by svenreppin** [comment]
it was my fault! sorry for any inconvenience! it was my fault, i've found my mistake. after checking `/var/log/auth.log` i saw a problem with my private key :-( thank you very much again! i love your tool!!! :-)

**Reply by isontheline** [reply]
You're Welcome 

Thank you so much for your support about WebSSH 🙏

**Comment by isontheline** [comment]
Hi 👋

Could you explain me more?
You can already use a PKEY without any password or passphrase.

Are you encountering any error?

Have a nice day ☀️

**Reply by svenreppin** [reply]
hi, yes, i am able to use a private key only. as mentioned it was a mistake on my side. greetings!
